jwst/outlier_detection/outlier_detection_step.py

```diff
@@ -142,7 +148,7 @@ def process(self, input_data):
             elif exptype in IFU_SPEC_MODES:
                 # select algorithm for IFU data
                 detection_step = outlier_registry['ifu']
-                pars['resample_suffix'] = 's3d'
+                pars['resample_suffix'] = 'crf'
```

Comment:
I'm also puzzled by this. Previously the s3d suffix was defined for attaching to the files created by "resampling" each input cal file into a cube, during the process of doing IFU outlier detection. But the new method no longer creates any resampled products while doing outlier detection right?  So I'm guessing this line could actually just be deleted, because it's no longer needed?